tests/system/model_monitoring/test_app.py

```diff
@@ -1089,44 +1091,76 @@ def _log_model(
             label_column=label_column,
         )
 
-    @classmethod
-    def _deploy_model_serving(
-        cls,
+    def _deploy_model_router(
+        self,
         name: str,
+        enable_tracking: bool = True,
+    ) -> mlrun.runtimes.nuclio.serving.ServingRuntime:
+        serving_fn = mlrun.code_to_function(
+            project=self.project_name,
+            name=name,
+            filename=f"{str((Path(__file__).parent / 'assets').absolute())}/models.py",
+            kind="serving",
+        )
+        serving_fn.set_topology("router")
+        for model_name, model_dict in self.router_models.items():
+            self._log_model(
+                model_name=model_name,
+                training_set=model_dict.get("training_set"),
+                label_column=model_dict.get("label_column"),
+            )
+            serving_fn.add_model(
+                model_name,
+                model_path=f"store://models/{self.project_name}/{model_name}:latest",
+                class_name=model_dict.get("class_name"),
+            )
+        serving_fn.set_tracking(enable_tracking=enable_tracking)
+        if self.image is not None:
+            serving_fn.spec.image = serving_fn.spec.build.image = self.image
+
+        serving_fn.deploy()
+        return typing.cast(mlrun.runtimes.nuclio.serving.ServingRuntime, serving_fn)
+
+    def _deploy_model_serving(
+        self,
         model_name: str,
         class_name: str,
         enable_tracking: bool = True,
         **kwargs,
```

Comment:
not in use
```suggestion
```

Reply:
I know that it un used but I still need it because I sending large dict with irrelevant param for this function.